src/locator/test.ts

```diff
@@ -65,7 +65,7 @@ describe("locate", () => {
 
   it("locates a damaged QR code and guesses the finder pattern location", async () => {
```

Comment:
Don't feel too strongly but wouldn't hurt to add a test case for this - Would be as simple as taking one of the originally failing ones and committing it into the locator tests.

Reply:
a52cda5